File: minimum-number-of-operations-to-move-all-balls-to-each-box/minimum-number-of-operations-to-move-all-balls-to-each-box.js
/**
 * @param {string} boxes
 * @return {number[]}
 */
var minOperations = function(boxes) {
  const result = [];
  for (let i = 0; i < boxes.length; i++) {
    let count = 0;
    for (let j = 0; j < boxes.length; j++) {
      if (Number(boxes[j]) === 1 && i !== j) {
        count += Math.abs(j - i); 
      }
    }
    result.push(count);
  }
  return result;
};

File: minimum-number-of-operations-to-move-all-balls-to-each-box/minimum-number-of-operations-to-move-all-balls-to-each-box.py
class Solution:
    def minOperations(self, boxes: str) -> List[int]:
      res = [];
      for i in range(len(boxes)):
        count = 0
        for j in range(len(boxes)):
          if(int(boxes[j]) == 1 and i != j):
            count += abs(i - j)
        res.append(count)
      return res